kolena/_experimental/quality_standard.py

```diff
@@ -91,3 +98,58 @@ def download_quality_standard_result(
         for model in models:
             result_dfs.append(_download_quality_standard_result(dataset, [model], metric_groups, intersect_results))
         return pd.concat(result_dfs, axis=1)
+
+
+@dataclass(frozen=True)
+class CopyQualityStandardRequest:
+    dataset_id: int
+    source_dataset_id: int
+    include_metric_groups: bool = True
+    include_test_cases: bool = True
+
+
+@with_event(event_name=EventAPI.Event.COPY_QUALITY_STANDARD_FROM_DATASET)
+def copy_quality_standards_from_dataset(
+    dataset: str,
+    source_dataset: str,
+    include_metric_groups: bool = True,
+    include_test_cases: bool = True,
+) -> Tuple[List[Dict[str, Any]], List[Dict[str, Any]]]:
+    """
+    Create quality standards on a dataset by copying from a source dataset. Note that this operation will overwrite the
+    existing quality standards on the dataset if they exist.
+
+    :param dataset: The name of the dataset.
+    :param source_dataset: The name of the dataset from which the quality standards should be copied.
+    :param include_metric_groups: Optional flag to indicate whether to copy the metric groups from the source dataset.
+    :param include_test_cases: Optional flag to indicate whether to copy the test cases from the source dataset.
+    :return: A tuple of the created metric groups and test cases.
+    """
+    if dataset == source_dataset:
+        raise IncorrectUsageError("source dataset and target dataset are the same")
+
+    if not include_test_cases and not include_metric_groups:
+        raise IncorrectUsageError("should include at least one of metric groups or test cases")
```

Comment:
`at least one of metric group or test case` instead of `at least one of metric groups or test cases`